internal/docs/pubsub/design.md

```diff
@@ -535,11 +471,19 @@ func (s *Subscription) Close() error {
 // tune sending and receiving of acks and messages. Behind the scenes,
 // NewSubscription spins up a goroutine to gather acks into batches and
 // periodically send them to the server.
-func NewSubscription(s driver.Subscription, opts SubscriptionOptions) *Subscription {
+func NewSubscription(s driver.Subscription) *Subscription {
 	// Details similar to the body of NewTopic should go here.
 }
 ```
 
+Topics will gather messages into batches for sending, using the
+existing [bundler](https://godoc.org/google.golang.org/api/support/bundler)
```

Comment:
It would be helpful to readers unfamiliar with bundler to include some explanation of what it is that bundler does (gather messages into batches based on time and space limits).

Reply:
@jba is saying he may do the batching with some bespoke code that doesn't use `bundler`. Removing the mention.